boards/imxrt1050-evkb/src/boot_header.rs

```diff
@@ -0,0 +1,412 @@
+/// Vector which contains the FCB header
+/// This section is extracted with objcopy from the hello world
+/// example in the i.MX RT 1050 EVKB SDK examples.
+pub const BOOT_HDR: [u8; 8192] = [
```

Comment:
Isn't it possible to define the section of this directly in rust, without needing an assembly file?

Reply:
For the moment that is the only solution that I managed to implement. I am working with someone from NXP in order to see how to write to code for it, however it is really complicated and will take a lot of time to write.